src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -35,6 +35,19 @@ export const presentableFormIDs = Object.keys(FORM_BENEFITS).reduce(
   {},
 );
 
+const idArray = MY_VA_SIP_FORMS.map(item => item.id);
+
+export const presentableFormIDsV2 = idArray.reduce((prefixedIDs, formID) => {
+  if (formID === VA_FORM_IDS.FEEDBACK_TOOL) {
+    prefixedIDs[formID] = 'FEEDBACK TOOL'; // eslint-disable-line no-param-reassign
```

Comment:
if we wanted to remove the `eslint-disable-line` comment, another way to write this might be to use a shallow copy of the Object, and shift labeling logic a bit, e.g.
```
var labelFormId = id => id; // TODO: update to do same "switch by ID" logic in own function

idArray.reduce((prefixedIDs, formID) => {
  const ids = { ...prefixedIDs, [formID]: labelFormId(formID), };
  // ...
  // console.log(ids);
  return ids;            
}, {});
```